recipes/flecs/all/conandata.yml

```diff
@@ -0,0 +1,10 @@
+sources:
+  "2.3.1":
+    url: "https://github.com/SanderMertens/flecs/archive/v2.3.1.tar.gz"
+    sha256: "b2cd69b3282167295d509deb762e580ad76040ed96ceb4492657bc1e444b8c89"
+patches:
+  "2.3.1":
+    - patch_file: "patches/less-warnings.patch"
```

Comment:
looking at the patch and the conversation it does not seem to be required..

I think patch could be removed

Reply:
> 
> 
> The commit/patch name is misleading. It removes invalid compiler warnings...